testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/SigningInTest.java

```diff
@@ -62,6 +64,9 @@
 /**
  * @author Vaclav Muzikar <vmuzikar@redhat.com>
  */
+@EnableFeature(value = Profile.Feature.ACCOUNT2, skipRestart = true)
+@EnableFeature(value = Profile.Feature.ACCOUNT_API, skipRestart = true)
```

Comment:
Is there any reason why to put here the account profile features? Seems a bit redundant, there are already present in the parent class.

Reply:
AFAIK, the problem is with `KeycloakContainerFeaturesController`, which manage and prepare all environments to enabling/disabling feature. The controller enable the feature only for specific class, therefore, it doesn't allow inheritance of the enabling/disabling feature from parent. So, the `account` features should be again explicitly specified.

I don't know, if it's a good approach, but it's not "target" of this PR :)

Reply:
Maybe @mposolda knows more details.

Reply:
Ok, makes sense. Even though I don't like this redundancy too much, we don't have much options right now. And rewriting `KeycloakContainerFeaturesController` is really out of scope of this PR.

Created [KEYCLOAK-13078](https://issues.redhat.com/browse/KEYCLOAK-13078) for that.